src/h5web/vis-packs/core/line/models.ts

```diff
@@ -15,3 +16,8 @@ export const GLYPH_URLS = {
   Square: squareURL,
   Cap: capURL,
 };
+
+export interface MappedArray {
+  base: ndarray | undefined;
+  mapped: ndarray | undefined;
+}
```

Comment:
The new interface returned by `useArrays`.

`MappedArray` is **again** not an ideal name